integration/src/integration/java/pl/allegro/tech/hermes/integration/HermesClientPublishingTest.java

```diff
@@ -47,6 +51,38 @@ public void shouldPublishUsingRestTemplate() {
         runTestSuiteForHermesClient(client);
     }
 
+    @Test
+    public void shouldPublishUsingOkHttpClient() {
+        // given
+        HermesClient client = hermesClient(new OkHttpSender()).withURI(topicURI).build();
+
+        // when & then
+        runTestSuiteForHermesClient(client);
+    }
+
+    @Test
+    public void shouldPublishUsingOkHttpClientUsingSSL() {
```

Comment:
The idea was that we check that everything works (by everything I mean entire test suite for HermesSender implementation) when we turn SSL on and in the other test we specifically check that connection was via h2 because that requires entirely different test case...